rasterio/crs.py

```diff
@@ -54,10 +57,15 @@ def from_string(prjs):
     if '{' in prjs:
         # may be json, try to decode it
         try:
-            return json.loads(prjs, strict=False)
+            val = json.loads(prjs, strict=False)
         except ValueError:
             raise ValueError('crs appears to be JSON but is not valid')
```

Comment:
@perrygeo What do you think about this `ValueError` for a bad JSON string?  We raise a `CRSError` everywhere else, and while this is technically a slightly different problem, the intended use and consequences are the same: the user has a string they _think_ contains a CRS they would like to decode.  I'm wondering if this shouldn't be a `CRSError` as well?  An malformed JSON string is also a malformed CRS.